src/wpool_queue_manager.erl

```diff
@@ -326,17 +380,18 @@ handle_call(
     true ->
       inc_pending_tasks(),
       { noreply
-      , State#state{clients = queue:in({Client, Event, Expires}, Clients)}
+      , State#state{clients =
+          queue:in({sync_send_all_event, Client, Event, Expires}, Clients)}
       };
     false ->
       {Worker, NewWorkers} = gb_sets:take_smallest(Workers),
       %NOTE: It could've been a while since this call was made, so we check
       case erlang:is_process_alive(ClientPid) andalso
         Expires > now_in_microseconds() of
         true  ->
-          Reply = wpool_fsm_process:sync_send_all_state_event(Worker, Event),
-          gen_server:reply(Client, Reply),
-          {noreply, State#state{workers = NewWorkers}};
+          NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),
```

Comment:
Line 392 is too long:           NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),.